File: javaEpam/Lab7/src/Aligator.java
public class Aligator extends Reptile {
    private int maxSpeed;
    private String skinTone;


    public Aligator(String name, String size, int weight, int maxSpeed, String skinTone) {
        super(name, size, weight);
        this.maxSpeed = maxSpeed;
        this.skinTone = skinTone;
    };

    public String getHeaders() {
        return super.getHeaders() + ',' + "maxSpeed";
    }

    public String toCSV() {
        return super.toCSV() + ',' + this.maxSpeed;
    }
}

File: python/lab_11/aligator.py
from crocodile import Crocodile

class Aligator(Crocodile):
  max_speed = 0

  def __init__(self, name=None, size=None, weight=None, skinTone=None, max_speed=None):
    super().__init__(name, size, weight, skinTone)
    self.max_speed = max_speed or 0
    
  def printInfo(self):
    print("Name ", self.name, "\nSize: ", self.size, "\nWeight: ", self.weight, "\nskinTone: ", self.skinTone, "\nMax speed: ", self.max_speed)